easyunfold/cli.py

```diff
@@ -165,9 +166,9 @@ def unfold_status(ctx):
     nkpts_sc = len(unfoldset.expansion_results['reduced_sckpts'])
     click.echo()
     click.echo(f'No. of k points in the primitive cell           : {unfoldset.nkpts_orig}')
-    click.echo(f'No. of expanded kpoints to be calculated cell   : {nkpts_sc} ({unfoldset.nkpts_expand})')
-    click.echo(f'No. of rotations in the primitive cell          : {unfoldset.pc_opts.shape[0]}')
-    click.echo(f'No. of rotations in the super cell              : {unfoldset.sc_opts.shape[0]}')
+    click.echo(f'No. of (non-symmetry-reduced) supercell kpoints : {nkpts_sc} ({unfoldset.nkpts_expand})')
```

Comment:
```suggestion
    click.echo(f'No. of supercell kpoints : {nkpts_sc}')
```

I realised that it is probably not much point to show `nkpts_expand`, which is the number of primitive cell k-points expanded by primitive cell symmetry, reduced by supercell symmetry, but before being transformed to supercell and deduplicated 😂